src/odemis/acq/stream/_sync.py

```diff
@@ -2322,8 +2322,22 @@ def _adjustHardwareSettings(self):
         # TODO: check that no fuzzing is requested (as it's not supported and
         # not useful).
 
+        # If a detector is "independent", then connect its dwell time
+        # Note: resolution is updated later, just before the acquisition block
+        # TODO: should it be done automatically by the "SettingsStream" in .prepare() or .linkHwVA()?
```

Comment:
When are we going to do all the TODO's? :smile:

Reply:
"One day". Well, for this TODO, I did add a task to the backlog, so this might really happen eventually!
https://delmic.atlassian.net/browse/CSDM-1131